projects/arlas-components/src/lib/components/mapgl/mapgl.component.ts

```diff
@@ -707,6 +707,14 @@ export class MapglComponent implements OnInit, AfterViewInit, OnChanges, OnDestr
     if (this.minZoom === undefined || this.minZoom === null) {
       this.maxZoom = 0;
     }
+    if (typeof this.defaultBasemapStyle.styleFile === 'string') {
+      this.defaultBasemapStyle.url = this.defaultBasemapStyle.styleFile;
+    }
+    this.basemapStyles.forEach(bm => {
+      if (typeof this.defaultBasemapStyle.styleFile === 'string') {
```

Comment:
Shouldn't it be typeof bm.styleFile === 'string' ?

Reply:
Absolut